Copy edit pass (#73)
ember-learn/super-rentals-tutorial: src/chapters/00-part-1.md

Welcome to the Ember Tutorial!

> In this tutorial, we will use Ember to build a professional looking application called Super Rentals. It's a website for browsing interesting places to stay for your next vacation. Check out the [finished app](https://super-rentals-tutorial--ember-super-rentals.netlify.com) to get a sense of the scope of the project.

Reviewer: ```suggestion
In this tutorial, we will use Ember to build an application called Super Rentals. This will be a website for browsing interesting places to stay for your next vacation. Check out the [finished app](https://super-rentals-tutorial--ember-super-rentals.netlify.com) to get a sense of the scope of the project.
```
